docs/auth.md

```diff
@@ -43,26 +232,23 @@ aggregates them into their respective files in `$HOME`.
        tekton.dev/git-0: github.com # Described below
    type: kubernetes.io/ssh-auth
    data:
-     ssh-privatekey: <base64 encoded>
+     ssh-privatekey: <base64-encoded>
      # This is non-standard, but its use is encouraged to make this more secure.
      # If it is not provided then the git server's public key will be requested
      # when the repo is first fetched.
-     known_hosts: <base64 encoded>
+     known_hosts: <base64-encoded>
```

Comment:
Change base64-encoded to just known_hosts

Reply:
Done.